### ISSUE TITLE
The Google-EteSync Add-on is a Trojan Horse: create a google integration add-on

### ISSUE THREAD

**Issue by piu-logistics** [etesync/etesync-web#169]
Please create a google plug-in that captures every time you send an email to a new contact.  Instead of just creating a new contact in google which is automatic the plug-in would prompt to store the contact in Ete in the correct group that you can select.  That would be useful to many, this add-on acts as a bridge to Ete, a Trojan Horse.

I get that the idea of EteSync is to get away from Google yet there's a process to moving and we have to meet our clients where they are, and help them on their way to where we know best, is best.  Ete does not capture the email facility, which is not its point, yes.  It is worth examining every connection G mail has with G cal and G contacts, and tactically cut each bind with a good solution / integration / pathway out.  We (your client market) will appreciate you (EteSync) for this. I would guess that 95% of the clients who would benefit from Ete and would love to use it are too deeply bound into the G, your project/product/service is in a unique position to give them a hand and solve their problems.  

Please do have a look at this:
https://developers.google.com/gsuite/add-ons/how-tos/testing-gsuite-addons?authuser=3#install_an_unpublished_add-on

I am able to assist with the audit of the connection, mentioned above.  Addressing them will make EteSync unstoppable (and more accessible to the non technical).

**Comment by piu-logistics** [comment]
Still very busy Tom?

I've been looking for solutions to this, please contact [GioBonvi](https://github.com/GioBonvi) about the feature request / issue i have raised, he can help:
https://github.com/GioBonvi/GoogleContactsEventsNotifier/issues/175#issue-731416786

**Comment by tasn** [comment]
Yeah, won't be able to get it to it in the near future, though I'll try. There are quite a few more important tasks on my queue atm.

**Comment by piu-logistics** [comment]
I see your situation, this is why i am looking for others who might be able to give a hand or lead on this internation point.  Please remain open to GioBonvi, his project seems to have the experience and established code base to address the issue / opportunity i have raised.

**Comment by tasn** [comment]
Yeah, that's great.
